samples/subsys/fs/littlefs/prj.conf

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2019 Peter Bigot Consulting, LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# Optionally force the file system to be recreated
+#CONFIG_APP_WIPE_STORAGE=y
+
+# littlefs needs strspn
+CONFIG_NEWLIB_LIBC=y
+
+# fs_dirent structures are big.
+CONFIG_MAIN_STACK_SIZE=2048
+
+CONFIG_LOG=y
+#CONFIG_FS_LOG_LEVEL_DBG=y
```

Comment:
remove commented line

Reply:
It's there as a hint so people don't waste time trying to set `CONFIG_FILE_SYSTEM_LOG_LEVEL_DBG=y` to figure out what's going wrong.

Reply:
ok, maybe worth to add appropriate comment (up to you)